### ISSUE TITLE
Some items of the message context menu are unreadable in the dark mode 

### ISSUE THREAD

**Issue by feimosi** [thunderbird-conversations/thunderbird-conversations#1790]
* **Thunderbird version:** 91.6.0 (64-bit)
* **Thunderbird Conversation version:** 4.0.11
* **Operating System:** Linux

If Thunderbird has the native dark mode turned on, then the message context menu renders in a weird way. Every other element has a dark background and is unreadable. I'd rather have them all white as in the light mode than like this.

In dark mode:
![image](https://user-images.githubusercontent.com/5721520/153617819-d2c3d011-9ba0-419a-aea4-383af363752c.png)

In light mode:
![image](https://user-images.githubusercontent.com/5721520/153618077-17941479-76a4-4d16-9dfe-05a097958f68.png)

**Comment by feimosi** [comment]
After some digging, I found out that the problem lies here:
https://github.com/thunderbird-conversations/thunderbird-conversations/blob/4b475f40191dd6b1bd7d41dbd04bec4ab5ff550f/addon/content/conversation.css#L325
Only `background-color` is set, without `color`. It's safer to just remove this line, then it'll render with light colors.

On hover it gets even darker:
![image](https://user-images.githubusercontent.com/5721520/153924421-2bdb3ea4-76ef-4ab9-93e7-82231cc38326.png)

However, there is a related issue, I'm not sure where it comes from but setting the `color` is ignored, even if I set it to `white`, it's still rendered black, so I just overrode it to use light background colors.
